README.md

```diff
@@ -3,19 +3,17 @@
 
-## Bit Camps Repository
+## Introducing Bit Camps
 
-This repository includes folders of curriculum for each of our Bit Camps. Our Bit Camps consist of weekly livestreams and homework assignments and conclude with a final project. Student volunteers act as livestream hosts and TAs to teach the curriculum and conduct office hours. For those of our Bit Camps sponsored by industry partners, a developer advocate often co-hosts the livestream and engineers serve as mentors to help students understand concepts and complete their final projects. Although mentorship and feedback on homeworks/final projects are only available to students who apply and get into the bootcamp, the curriculum is free and open to all.
+Bit Camp consists of weekly live lectures (hosted by student developers), assignments, office hours, and the final project. Although mentorship and feedback are only available to students who apply and get into the boot camp, the curriculum is FREE and OPEN to all who want to learn new technologies and use their skills to their side projects. 
 
-In the past, our curriculum has been created by student developers, sometimes in collaboration with industry partners for our sponsored Bit Camps. As an open source organization, we want to increase community involvement and encourage more developers to contribute to our content. Each Bit Camp directory includes a readme describing the program syllabus, as well as folders for livestreams and homeworks. If you think any of our existing content is confusing to understand for beginners, too easy, too hard, etc., open a PR and let us know! We welcome all suggestions and edits to our repo.
+To find out more, click on the links below. Additionally, we also provide a syllabus in the readme section to better understand each curriculum.
```

Comment:
I like that this is concise, but try to add in these two ideas to the description:
* As an open source organization, we want to increase community involvement and encourage more developers to contribute to our content.
* If you think any of our existing content is confusing to understand for beginners, too easy, too hard, etc., open a PR and let us know! We welcome all suggestions and edits to our repo.

_Also, make sure "readme section" is changed to README.md_